doc/gcylcrc.tex

```diff
@@ -12,6 +12,31 @@ \section{Gcylc Config File Reference}
 
 \subsection{Top Level Items}
 
+\subsubsection{dot icon size}
+
+Set the size of the task state dot icons displayed in the text and dot
+views.
+
+\begin{myitemize}
+\item {\em type:} string
+\item {\em legal values:} ``small'' (10px), ``medium'' (14px), ``large'' (20px)
+\item {\em default:} ``medium''
+\end{myitemize}
+
+
+\subsubsection{initial side-by-side views}
+
+Set the suite view panels initial orientation when the GUI starts.
+This can be changed later using the "View" menu "Toggle views side-by-side"
+ option.
+
+\begin{myitemize}
+\item {\em type:} boolean (False or True)
+\item {\em legal values:} ``False'', ``True''.
+\item {\em default:} ``False''
+\end{myitemize}
```

Comment:
For consistency with other boolean items: we aren't giving "legal values" lines (it should be obvious), and we aren't quoting the default value.